scheduler/hooks/retrieve_after_store.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+# Usage: bash retrieve_after_store.sh <UUID>
+# expected installed env binaries: mktemp, curl, jq
+# env vars used:
+# DEALBOT_CONTROLLER_ENDPOINT
+
+curl -o /tmp/dealbot.$1 $DEALBOT_CONTROLLER_ENDPOINT/tasks/$1\?parsed
```

Comment:
pretty sure you don't need to escape the question mark FYI, it's not at all special in terms of shell syntax